src/main/views/search.njk

```diff
@@ -1,48 +1,79 @@
 {% extends "template.njk" %}
 {% from "macros/common-components.njk" import goBack, submitButton %}
-{% from "govuk/components/input/macro.njk" import govukInput %}
+{% from "govuk/components/error-summary/macro.njk" import govukErrorSummary %}
+
 {% block pageTitle %}
     {{ "Search" | safe }}
 {% endblock %}
 
 {% block beforeContent %}
-  {{ goBack('/search-option') }}
+    {{ goBack('/search-option') }}
 {% endblock %}
 
 {% block content %}
-  <form action="search" method="post">
-    <div class="govuk-form-group govuk-!-width-one-half">
-      <h1 class="govuk-label-wrapper"><label class="govuk-label govuk-label--l" for="event-name">
-          What court or tribunal are you interested in?
-        </label>
-      </h1>
-      <div id="event-name-hint" class="govuk-hint">
-        For example Blackburn Family Court or Chester
-      </div>
-      <div class="govuk-!-width-two-thirds" id="search-input-container"></div>
-    </div>
-
-
-    {{ submitButton() }}
-  </form>
+    {% if noResultsError or invalidInputError %}
+        {% set errorMessage = 'There is nothing matching your criteria' if noResultsError else 'Search must be three characters or more' %}
+        <div class="govuk-grid-row">
+            <div class="no_padding govuk-grid-column-three-quarters">
+                {{ govukErrorSummary({
+                    titleText: "There is a problem",
+                    errorList: [
+                        {
+                            text: errorMessage,
+                            href: "search-input-container"
+                        }
+                    ]
+                }) }}
+            </div>
+        </div>
+    {% endif %}
+    {% set inputClasses = 'govuk-!-width-two-thirds govuk-input--error'
+        if noResultsError or invalidInputError else 'govuk-!-width-two-thirds' %}
+    {% set formClasses = 'govuk-form-group--error govuk-form-group govuk-!-width-one-half'
+        if noResultsError or invalidInputError else 'govuk-form-group govuk-!-width-one-half' %}
+    <form action="search" method="post">
+        <div class="{{ formClasses }}">
+            <h1 class="govuk-label-wrapper">
+                <label class="govuk-label govuk-label--l" for="search-input">
+                    What court or tribunal are you interested in?
+                </label>
+            </h1>
+            <div id="search-input-hint" class="govuk-hint">
+                For example Blackburn Family Court or Chester
+            </div>
+            {% if noResultsError or invalidInputError %}
+                <span class="govuk-error-message">Enter a court name, town or city</span>
```

Comment:
Just on the error message - the user flow in the story says it should say 'Please enter a court or tribunal', however in the screenshots the above is correct (Enter a court name, town or city).

Is the above the correct one to be displayed?

Reply:
I was led by the screenshots from the ticket attachments

Reply:
I was led by screenshots from the ticket attachments

Reply:
Junior confirmed it should be Enter a court name, town or city, updated the ticket accordingly